model.py

```diff
@@ -224,6 +276,28 @@ def forward_pass_test():
         im.save(f'pred_img_{i+1}.png')
 
 
+if __name__ == '__main__':
+    input_str = "test string"
+    # Tested with random small AutoencoderKL initialized as follows:
+    # torch.manual_seed(0)
+    # vae = AutoencoderKL(
+    #     block_out_channels=[32, 64],
+    #     in_channels=3,
+    #     out_channels=3,
+    #     down_block_types=["DownEncoderBlock2D", "DownEncoderBlock2D"],
+    #     up_block_types=["UpDecoderBlock2D", "UpDecoderBlock2D"],
+    #     latent_channels=4,
+    # )
+    test_model = LLourney(
+        image_dim=64,
+        # vae_model_id_or_path="test_vae",
```

Comment:
Do we still need the "test_vae" you defined above around for the test? Or is it the default 'runwayml/stable-diffusion-v1-5' small enough?

Reply:
I guess the current `LLourney` parameters for the test are meant to work for a small VAE; I believe if you use a 512 x 512 image and the VAE from `runwayml/stable-diffusion-v1-5`, the patch embedding (with `patch_dim=2`) will produce way more tokens than the `hf-internal-testing/tiny-random-gpt2` testing checkpoint's context window can handle. So it's probably best to stick with `test_vae`, and it makes the test faster and less resource-intensive to run since it's smaller. (I guess it could be worth uploading it to the hub so we don't have to keep around the code to reproduce it.)

Reply:
Sounds good to use it for the test, but then should we uncomment out the code that creates/saves it? Maybe with a check to see whether it has already been saved.

Reply:
Added a simple check to see if the `test_vae` directory exists and if not generate the random small VAE checkpoint for testing.